config/registry/common/modules/mongodb-atlas.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Atlas Mongo"
-linkTitle: "Atlas Mongo"
+title: "Mongdb Atlas"
```

Comment:
`title` and `linkTitle` need to be the same as the module name: `mongodb-atlas`

Reply:
oh ok, I didnt realize that. will fix